src/helix-ui/styles/components/pills.less

```diff
@@ -7,6 +7,7 @@ hx-status {
   background-color: transparent;
   border: 1px solid @gray-750;
   border-radius: 1em; // 10px
+  box-sizing: border-box;
```

Comment:
Please revert. We already handle this in `reset.less` 

https://github.com/rackerlabs/helix-ui/blob/master/src/helix-ui/styles/reset.less#L14-L26

Reply:
Without this line, this is the `<hx-status>` behavior: 

![image](https://user-images.githubusercontent.com/10120600/56837401-e70b0c00-683f-11e9-92a0-9f2e45d05f27.png)

Reply:
Huh... I'll have to take a gander under the hood.

Reply:
See https://github.com/rackerlabs/helix-ui/pull/488#discussion_r279112082